demos/notebooks/demo_pipeline.ipynb

```diff
@@ -610,10 +1023,736 @@
+    "\n",
+    "    movie_path1 = download_demo('Sue_Split1.tif')\n",
+    "    movie_path2 = download_demo('Sue_Split2.tif')\n",
+    "    movie_paths = [movie_path1, movie_path2]\n",
+    "\n",
+    "Then, when creating the movie object instead of using `cm.load()` you would use `cm.load_movie_chain()` which takes in a list as an argument:\n",
+    "\n",
+    "    movie_orig = cm.load_movie_chain(movie_paths)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "   \n",
+    "If you have <b>noncontiguous</b> recording sessions, for instance data in files from sessions separated by many days or weeks, and you need to register/match the neurons from these sessions, this is a different use case. We do have a demo for that: see [demo_multisession_registration.ipynb](./demo_multisession_registration.ipynb).\n",
+    "    \n",
+    "</div> \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=display_large></a>\n",
+    "## Appendix 3: Displaying large files\n",
+    "The way we have loaded movie objects in the main demo requires loading all of the data you want to view into memory, and this is not going to be possible with extremely large data sets. But even with very large data sets, you typically want to visualize what is going on, make sure things seem reasonable, etc. Caiman has built-in tools to handle this case. One, you just load *some* of a movie into a movie object using the `subindices` argument to the `load()` function. For example, if you just want to load the first 500 frames of a movie, you can send it `subindices=np.arange(0,500)`. \n",
+    "\n",
+    "Probably even better, there is a `play_movie()` function that behaves just like `movie.play()` in the main demo, that doesn't ever load the movie into memory, but just takes the filename as an argument and iteratively loads frames from disk and shows them as needed. If you want to play with it, just import with `caiman.base.movies import play_movie` and read the documentation. We don't use it for the demo because the demo movie is small, and we actually do some calculations on the loaded movie array in the demo. This may change in the future. \n",
+    "\n",
+    "Another option for viewing very large movies is to use the [fastplotlib](https://github.com/fastplotlib/fastplotlib) library, which leverages GPU-based visualization. Like `play_movie()`, it uses lazy loading so will not overload your memory, and provides interactive Jupyter widgets to explore your movies. "
```

Comment:
Not sure it's helpful to mention this if it's not trivial for users to pick it up and use it for this purpose.

Reply:
What useful would users get out of this? That there's some graphing library out there that if users spend their time learning they might be able to create their own visualisations with some development effort? Is that a useful thing to say? Would we say that about, say, gtk, qt, or SDL? Consider:

Another option to visualise your data with cool 3d effects is the Unity graphics engine. It efficiently uses GPU-based visualization, saving your CPU for computation and intelligently deciding what to render, saving memory.

After reading that little advert, same as with fpl, the user is informed about something that they're not going to directly find usable.

Reply:
If we want to point them at mescore or something that wraps caiman that uses FPL, that's fine. That's something the user can explore on their own without coding. It's a concrete suggestion, the response isn't just "oh".

Reply:
Would be useful to hear what @kushalkolar thinks on this, but just to push back a bit. 

I get your point, but the difference is Qt etc don't let you quickly visualize your work in Jupyter in an interactive way. We are mentioning a tool specifically because it will be useful and save them time in the long run, for the types of data they are analyzing, in the medium they are doing it (I see our job to help them not torture them by mentioning Blender or Qt  :smile:) . I'm ok mentioning things that require people doing _some_ work.  In that sense it's an advert but seems justified, like my mention of mpld3 elsewhere in the nb. We could even throw in minimal code example showing how to use an imagewidget.

Reply:
> more experienced programmers will be able to work with it

Um, the ImageWidget is already trivial to use, so I don't see what you mean here? 

```python
iw = fpl.ImageWidget(data)
iw.show()
```

I think it could make more sense to just have fastplotlib code cells in the notebook. [they won't work in colab yet](https://github.com/vispy/jupyter_rfb/pull/76), but nobody runs these in colab for their own work. 

One issue is that it's not possible to lazy-load every type of file format. It works with hdf5, zarr, etc. if the chunking is correct, and it works with _some_ tiff files. It works with all memmaps though, so that's an easy starting point.

Reply:
But easiest is mesmerize viz where they just have to call `df.iloc[ix].mcorr.viz()` or `df.iloc[ix].cnmf.viz()` and they get everything they could ever want.

Reply:
mesmerize-vis would be ideal if we're going to mention something.

Reply:
We need a landing page to describe the whole ecosystem, newcomers will be confused.

Reply:
This is the beginning of the nb discussing how to visualize your raw data (in this case a tiff file), and solutions for when your data is too big to use the standard `load()` functionality. fpl is more appropriate in this context that mesmerize-viz, but because there are wrinkles with handling tiff, it might be best just to hold off until later when I'm already mentioning mesmerize.

Reply:
Come to think of it, is there a reason to not just put in an ImageWidget now? Are we waiting for changes in fastplotlilb or anything?

Reply:
I can think of two main concerns: this would mean we'd be forcing people to jupyterlab instead of jupyter notebook (is this right @kushalkolar ?). Second, at least middle of the year my caiman movies were not always working in ImageWidget class, so would not work for viewing original raw data (sometimes it did work, sometimes not).

Reply:
> My main concern is that sometimes the caiman movie objects even when loaded, don't render properly in fastplotlib Image Widgets

This was long ago before auto vmin vmax